crates/core/action-pipeline/src/processor.rs

```diff
@@ -177,17 +177,9 @@ pub async fn process_action(
         }
     };
 
-    local_emitter
```

Comment:
This wasn't capturing the finished dates/final state, since it happened in the next block <_<